### PR TITLE
fixed CoC

### PR DESCRIPTION
**Pull request by Al-Caveman**
fixed CoC.

### CODE REVIEW

**Comment by AlmightyFloppyFish** [comment]
How to delete someone else's pull request?

**Comment by thibmo** [comment]
Not.

**Comment by lucaisacoder** [comment]
Good job



在 2018年10月5日，10:43，Toraboran Caveman <notifications@github.com<mailto:notifications@github.com>> 写道：


fixed CoC.

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/torvalds/linux/pull/604

Commit Summary

  *   Fixed CoCk bug
  *   fixed typo in updated CoC.

File Changes

  *   M Documentation/process/code-of-conduct.rst<https://github.com/torvalds/linux/pull/604/files#diff-0> (91)

Patch Links:

  *   https://github.com/torvalds/linux/pull/604.patch
  *   https://github.com/torvalds/linux/pull/604.diff

File: Documentation/process/code-of-conduct.rst [hunk 1]
```diff
@@ -1,81 +1,24 @@
...
-Our Standards
-=============
+  * Stay on-topic.  If you think someone is e-harassing your ass, please fuck off
+    and discuss it in pornhub.com or whatever trashbin.  We are not interested in
+    hearing lectures about e-sex and e-harassment.  We discuss how to make the 
+    kernel better. Period.
 
-Examples of behavior that contributes to creating a positive environment
-include:
+  * Only men can contribute.  A woman can contribute only after obtaining
```
Comment (protonesso) [inline Documentation/process/code-of-conduct.rst]: :ok_hand:

**Comment by Al-Caveman** [comment]
GitHub threatened to permanently delete my account if I did not close this Pull Request.  So I am closing this Pull Request.

Below are the messages from GitHub:

> Hello,
> 
> Your account was suspended following reports of a pull request opened in the torvalds/linux repository that is in violation of our Terms of Service prohibition against content that is,
> 
> > discriminatory or abusive toward any individual or group;
> 
> At this time, if you will agree to close the pull request we can reinstate your account and provide a direct link to the pull request in question. Please let us know how you'd like to proceed.
> 
> Regards,
> GitHub Staff

When I asked them _where in my Pull Request am I abusing or discriminating against others?_ they replied:

> Hello,
> 
> To be clear, the entire pull request is in violation of our terms of service and must be closed at this time. We have reinstated your account and given you 24 hours to comply.
> 
> If the pull request is not closed, or if we receive reports of further such violations of our terms, we may permanently suspend your account.
> 
> Regards, 
> GitHub Staff

Here is my thought:

- GitHub's CoC states that discrimination/abuse is not allowed.  But simply sharing ideas -by itself- does not _actually_ discriminate nor abuse any thing.  It's just a mere floating idea.
- In order to be able to actually discriminate/abuse others, one needs _power_ or _enforcement_.  I clearly lack power/enforcement.  Hence, I cannot actually discriminate/abuse anyone, no matter how many Pull Requests I submit.
- The only one who was discriminated/abused against in this case is _me_, where I was forced to not even share my ideas, simply because I belong to a minority group of  people that their culture differs than the culture of GitHub staff.
- It is very sad that GitHub considers the simple act of sharing ideas _discrimination_ or _abuse_ against people, but when GitHub actually discriminates and oppresses me from contributing my ideas, they don't find their oppression as _discrimination nor abuse_.

**Comment by Al-Caveman** [comment]
I am hereby closing in order to not get my account permanently deleted.

**Comment by kaisersGa** [comment]
24 hours to comply

At 2018-10-12 09:05:30, "Toraboran Caveman" <notifications@github.com> wrote:


GitHub threatened to permanently delete my account if I did not close this Pull Request. So I am closing this Pull Request.

Below are the messages from GitHub:

Hello,

Your account was suspended following reports of a pull request opened in the torvalds/linux repository that is in violation of our Terms of Service prohibition against content that is,

discriminatory or abusive toward any individual or group;

At this time, if you will agree to close the pull request we can reinstate your account and provide a direct link to the pull request in question. Please let us know how you'd like to proceed.

Regards,
GitHub Staff

When I asked them where in my Pull Request am I abusing or discriminating against others? they replied:

Hello,

To be clear, the entire pull request is in violation of our terms of service and must be closed at this time. We have reinstated your account and given you 24 hours to comply.

If the pull request is not closed, or if we receive reports of further such violations of our terms, we may permanently suspend your account.

Regards,
GitHub Staff

Here is my thought:

GitHub's CoC states that discrimination/abuse is not allowed. But simply sharing ideas -by itself- does not actually discriminate nor abuse any thing. It's just a mere floating idea.
In order to be able to actually discriminate/abuse others, one needs power or enforcement. I clearly lack power/enforcement. Hence, I cannot actually discriminate/abuse anyone, no matter how many Pull Requests I submit.
The only one who was discriminated/abused against in this case is me, where I was forced to not even share my ideas, simply because I belong to a minority group of people that their culture differs than the culture of GitHub staff.
It is very sad that GitHub considers the simple act of sharing ideas discrimination or abuse against people, but when GitHub actually discriminates and oppresses me from contributing my ideas, they don't find their oppression as discrimination nor abuse.

**Pull request closed without merging**